### DISCUSSION TITLE
Certificado Partners - Development

### DISCUSSION THREAD

**Discussion by GiseleZerbmaster** [rodurma/forum-meli#11] [Usuários e Aplicativos]
Bom dia! Alguém sabe me dizer se, para conseguir a certificação do Ml, é necessário ter clientes em um mesmo aplicativo? Porque hoje eu crio um aplicativo para cada cliente que tenho. Obrigada.

**Comment by MikeGBarreto** [comment]
Você tem que ter apenas 1 aplicativo para todos os clientes. Os requisitos para obter o certificado estão nesse link: https://developers.mercadolivre.com.br/pt_br/developer-partner-program
